glance/server.sls

```diff
@@ -79,6 +79,27 @@ glance_install_database:
   - require:
     - service: glance_services
 
+{%- if server.get('image_cache', {}).get('enabled', False) %}
+glance_cron_glance-cache-pruner:
```

Comment:
Well, assuming they have shared storage (which is not the default), it might result in filenotfound errors. Do you have a better solution in mind?